### DISCUSSION TITLE
Viztracer is great

### DISCUSSION THREAD

**Discussion by time4tea** [gaogaotiantian/viztracer#301] [General]
Just wanted to say viztracer is great. I had an inkling of a performance issue in my code, but didn't know how to prove it easily. Eventually discovered viztracer and boom - 2 mins later - amazing flamegraph demonstrating exactly the issue.

Here's a link to the graph i generated  https://github.com/time4tea/gopro-dashboard-overlay/blob/main/PERFORMANCE.md

I don't know if anybody else also wanted to share the stuff they had discovered. 

Anyhow, thanks for a nice thing.

Cheers!

james

**Comment by gaogaotiantian** [comment]
Hi! I'm glad that you liked VizTracer. I do hope that it's easy to use and helpful for your understanding of the program!

**Comment by david-macleod** [comment]
I second this. Have tried pretty much every Python profiling tool out there and VizTracer stands apart in terms of ease of integration + functionality. Would give the repo more than 1 star if I could 🌟

**Comment by wilderfield** [comment]
Stubled upon this, and WOW! Fantastic tool, great documentation, convenient packaging, easy to use, thanks @gaogaotiantian

**Reply by gaogaotiantian** [reply]
Thank you for the good words! Let others know LOL.
